tests/test_hubspot_all_fields_test.py

```diff
@@ -96,11 +98,20 @@ def testable_streams(self):
             'subscription_changes', # BUG_TDL-14938 https://jira.talendforge.org/browse/TDL-14938
         })
 
+    def get_properties(self):
+        return {'start_date' : '2021-08-05T00:00:00Z'}
+
+    # TODO move the overriden start date up as much as possible to minimize the run time
+    #      it can probably be dynamic like today minus 7 days
 
     @classmethod
     def setUpClass(cls):
         cls.maxDiff = None  # see all output in failure
 
+        # TODO my_timestamp needs to be driven off of start_date
```

Comment:
Oh we should do this so the run time does not grow each day.

Reply:
will do this when test and data is more stable